src/sentry/runner/commands/cleanup.py

```diff
@@ -332,7 +332,7 @@ def is_filtered(model):
         BulkDeleteQuery(
             model=models.EventMapping,
             dtfield='date_added',
-            days=min(days, 7),
+            days=days,
```

Comment:
We should just roll this into `BULK_QUERY_DELETES` now and remove this special block entirely.